Reddit Users not completely downloaded due to illegal chars in post titles.

* Ripme version: 2.1.3-18-ea5ffdba
* Java version: 
```
openjdk 14.0.2 2020-07-14
OpenJDK Runtime Environment (build 14.0.2+12-Ubuntu-120.04)
OpenJDK 64-Bit Server VM (build 14.0.2+12-Ubuntu-120.04, mixed mode, sharing)
```
* Operating system: `Linux pop-os 5.19.0-76051900-generic #202207312230~1663791054~22.04~28340d4 SMP PREEMPT_DYNAMIC Wed S x86_64 x86_64 x86_64 GNU/Linux` is the result of `uname -a`
<!-- Please do not link to content featuring underage characters even if the characters are drawn.
 These works are still illegal in many places including much of America -->
* Exact URL you were trying to rip when the problem occurred: https://www.reddit.com/user/Travis4687/
* Please include any additional information about how to reproduce the problem:

Open the application, rip the reddit users profile, close the application.

## Expected Behavior

The user's profile, all self posts and images linked that have rippers, should be downloaded successfully.

## Actual Behavior

Not all posts are downloaded successfully, namely posts with illegal filename characters in the title. A post titled "[USA-TX] [H] GTX 1070 / GTX 1660ti / i7-9700 / 32GB DDR4-2666 / 1TB SSD SATA [W] PayPal / Local Cash" is not downloaded due to the `/` in the title not being allowed in a filename.

App Log:
```
Downloading https://i.imgur.com/eH3FtPo.jpg
Downloading https://i.imgur.com/cme6hre.jpg
Downloading https://i.imgur.com/FJcR9Cf.jpg
Downloading https://i.imgur.com/9XfX2t4.jpg
Downloaded ../../..
Downloaded ../../../..
Downloaded ../../../..
Downloaded ../../../..
Downloading https://www.reddit.com/r/hardwareswap/comments/ibsq0d/usatx_h_dell_g3_gaming_laptop_16gb_ram_500gb_ssd/
Downloading https://www.reddit.com/r/hardwareswap/comments/iakdc0/usatx_h_dell_g3_gaming_laptop_16gb_ram_500gb_ssd/
Downloading https://i.imgur.com/UlKHCWC.jpg
Downloading https://i.imgur.com/8iXhaCI.jpg
Downloaded ../../../..
Downloading https://i.imgur.com/uwY3dMM.jpg
Downloaded ../../../..
Downloaded ../../../..
Downloading https://www.reddit.com/r/hardwareswap/comments/iakbcp/usatx_h_asus_nvidia_gtx_1650_4gb_w_paypal/
Downloading https://www.reddit.com/r/SecurityClearance/comments/beb6nu/timeline_on_incident_reports_in_jpas/
Downloading https://www.reddit.com/r/hardwareswap/comments/av8h4z/usatxh_evga_980ti_sc_w_paypal_local_cash/
Downloading https://www.reddit.com/r/hardwareswap/comments/av8djw/usatxh_evga_980ti_sc_w_paypal/
Downloading https://www.reddit.com/r/hardwareswap/comments/av89kz/usatxh_evga_980ti_sc_w_paypal/
Downloading https://i.imgur.com/kZkd14R.jpg
Downloaded ../../../..
Downloading https://www.reddit.com/r/Persona5/comments/6gxwxc/just_started_ng_confused_about_something/
Downloading https://www.reddit.com/r/KingdomHearts/comments/66mpk8/kh2_still_issues_with_the_collection_safe_to_buy/
Downloading https://www.reddit.com/r/KingdomHearts/comments/66mnol/kh125_is_the_collection_safe_to_buy_been_a_few/
Downloading https://www.reddit.com/r/hardwareswap/comments/66lmb1/usatx_w_core_i73770_h_paypal/
Downloading https://www.reddit.com/r/PaxPassExchange/comments/5qggz9/wtb_south_sat_or_sun_tickets/
Downloading https://i.imgur.com/6VktOQG.jpg
Downloading https://i.imgur.com/XjNFKQE.jpg
Downloading https://i.imgur.com/xkUvMqm.jpg
Downloaded ../../../..
Downloaded ../../../..
Downloaded ../../../..
Downloading https://i.imgur.com/mt5nmTd.jpg
Downloading https://i.imgur.com/hClJbVF.jpg
Downloaded ../../../..
Downloading https://i.imgur.com/BA0xIeJ.jpg
Downloaded ../../../..
Downloaded ../../../..
Downloading https://i.imgur.com/BBWUjrc.jpg
Downloading https://i.imgur.com/OnviMQi.jpg
Downloading https://i.imgur.com/UVepKHT.jpg
Downloading https://i.imgur.com/ma4j7ES.jpg
Downloaded ../../../..
Downloaded ../../../..
Downloaded ../../../..
Downloaded ../../../..
Downloading https://www.reddit.com/r/hardwareswap/comments/3j3igu/usatxh_asus_strix_gtx_960_oc_2gb_gddr5_w_paypal/
Rip complete, saved to /home/<user>/Downloads/rips/reddit_user_Travis4687
```

Terminal Log:
```
18:46:23.392 [main] INFO  com.rarchives.ripme.ui.MainWindow - Loading history from /home/nathan/.config/ripme/history.json
18:46:23.886 [Thread-1] INFO  com.rarchives.ripme.ui.UpdateUtils - Running version (2.1.3-18-ea5ffdba) is not older than release (2.1.2-23-e5438e85)
18:46:26.991 [AWT-EventQueue-0] INFO  com.rarchives.ripme.utils.Utils - Saved configuration to /home/nathan/.config/ripme/rip.properties
18:46:27.089 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for www.reddit.com
18:46:27.090 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for reddit.com
18:46:27.506 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper -     Retrieving https://imgur.com/a/DLQjJFL](https://imgur.com/a/DLQjJFL/all
18:46:27.507 [Thread-3] ERROR com.rarchives.ripme.utils.RipUtils - [!] Exception while loading album https://imgur.com/a/DLQjJFL](https://imgur.com/a/DLQjJFL
java.net.MalformedURLException: Illegal character in path at index 37: https://api.imgur.com/3/album/DLQjJFL](https://imgur.com
	at org.jsoup.helper.CookieUtil.asUri(CookieUtil.java:80) ~[ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at org.jsoup.helper.CookieUtil.applyCookiesToRequest(CookieUtil.java:41) ~[ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at org.jsoup.helper.HttpConnection$Response.createConnection(HttpConnection.java:1033) ~[ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at org.jsoup.helper.HttpConnection$Response.execute(HttpConnection.java:856) ~[ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at org.jsoup.helper.HttpConnection$Response.execute(HttpConnection.java:829) ~[ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at org.jsoup.helper.HttpConnection.execute(HttpConnection.java:366) ~[ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at org.jsoup.helper.HttpConnection.get(HttpConnection.java:353) ~[ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at com.rarchives.ripme.ripper.rippers.ImgurRipper.getAlbumData(ImgurRipper.java:346) ~[ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at com.rarchives.ripme.ripper.rippers.ImgurRipper.getImgurAlbum(ImgurRipper.java:257) ~[ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at com.rarchives.ripme.utils.RipUtils.getFilesFromURL(RipUtils.java:38) [ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at com.rarchives.ripme.ripper.rippers.RedditRipper.handleURL(RedditRipper.java:406) [ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at com.rarchives.ripme.ripper.rippers.RedditRipper.handleBody(RedditRipper.java:245) [ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at com.rarchives.ripme.ripper.rippers.RedditRipper.parseJsonChild(RedditRipper.java:219) [ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at com.rarchives.ripme.ripper.rippers.RedditRipper.getAndParseAndReturnNext(RedditRipper.java:116) [ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at com.rarchives.ripme.ripper.rippers.RedditRipper.rip(RedditRipper.java:90) [ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at com.rarchives.ripme.ripper.AbstractRipper.run(AbstractRipper.java:627) [ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at java.lang.Thread.run(Thread.java:832) [?:?]
Caused by: java.net.URISyntaxException: Illegal character in path at index 37: https://api.imgur.com/3/album/DLQjJFL](https://imgur.com
	at java.net.URI$Parser.fail(URI.java:2938) ~[?:?]
	at java.net.URI$Parser.checkChars(URI.java:3109) ~[?:?]
	at java.net.URI$Parser.parseHierarchical(URI.java:3191) ~[?:?]
	at java.net.URI$Parser.parse(URI.java:3139) ~[?:?]
	at java.net.URI.<init>(URI.java:623) ~[?:?]
	at java.net.URL.toURI(URL.java:1063) ~[?:?]
	at org.jsoup.helper.CookieUtil.asUri(CookieUtil.java:78) ~[ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	... 16 more
https://www.reddit.com/r/hardwareswap/comments/yaxdsw/usatx_h_gtx_1070_gtx_1660ti_i79700_32gb_ddr42666/
18:46:27.933 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for www.reddit.com
18:46:27.933 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for reddit.com
18:46:28.057 [Thread-3] ERROR com.rarchives.ripme.ripper.AbstractRipper - [!] Error creating save file path for description 'https://www.reddit.com/user/Travis4687/':
java.nio.file.NoSuchFileException: /home/nathan/Downloads/rips/reddit_user_Travis4687/yaxdsw_[USA-TX] [H] GTX 1070 / GTX 1660ti / i7-9700 / 32GB DDR4-2666 / 1TB SSD SATA [W] PayPal / Local Cash.html
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:92) ~[?:?]
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111) ~[?:?]
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:116) ~[?:?]
	at sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:219) ~[?:?]
	at java.nio.file.spi.FileSystemProvider.newOutputStream(FileSystemProvider.java:478) ~[?:?]
	at java.nio.file.Files.newOutputStream(Files.java:224) ~[?:?]
	at com.rarchives.ripme.ripper.rippers.RedditRipper.saveText(RedditRipper.java:297) [ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at com.rarchives.ripme.ripper.rippers.RedditRipper.getAndParseAndReturnNext(RedditRipper.java:123) [ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at com.rarchives.ripme.ripper.rippers.RedditRipper.rip(RedditRipper.java:90) [ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at com.rarchives.ripme.ripper.AbstractRipper.run(AbstractRipper.java:627) [ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at java.lang.Thread.run(Thread.java:832) [?:?]
18:46:28.058 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper -     Retrieving https://imgur.com/a/yvxKTnk/all
18:46:28.221 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper - {"data":{"id":"yvxKTnk","title":null,"description":null,"datetime":1644743446,"cover":"eH3FtPo","cover_edited":null,"cover_width":1908,"cover_height":4032,"account_url":null,"account_id":null,"privacy":"hidden","layout":"blog","views":51,"link":"https:\/\/imgur.com\/a\/yvxKTnk","favorite":false,"nsfw":false,"section":null,"images_count":1,"in_gallery":false,"is_ad":false,"include_album_ads":false,"is_album":true,"images":[{"id":"eH3FtPo","title":null,"description":null,"datetime":1644743444,"type":"image\/jpeg","animated":false,"width":1908,"height":4032,"size":210676,"views":42,"bandwidth":8848392,"vote":null,"favorite":false,"nsfw":null,"section":null,"account_url":null,"account_id":null,"is_ad":false,"in_most_viral":false,"has_sound":false,"tags":[],"ad_type":0,"ad_url":"","edited":"0","in_gallery":false,"link":"https:\/\/i.imgur.com\/eH3FtPo.jpg"}],"ad_config":{"safeFlags":["not_in_gallery","share"],"highRiskFlags":[],"unsafeFlags":["sixth_mod_unsafe"],"wallUnsafeFlags":[],"showsAds":false,"showAdLevel":1}},"success":true,"status":200}
18:46:28.244 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper - Writing https://i.imgur.com/eH3FtPo.jpg to file
18:46:28.250 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper -     Retrieving https://imgur.com/a/S2VbYTE/all
18:46:28.250 [pool-2-thread-1] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [!] Deleting existing file../../..
18:46:28.250 [pool-2-thread-1] INFO  com.rarchives.ripme.ripper.DownloadFileThread -     Downloading file: https://i.imgur.com/eH3FtPo.jpg Retry #1
18:46:28.324 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper - {"data":{"id":"S2VbYTE","title":null,"description":null,"datetime":1597543454,"cover":"cme6hre","cover_edited":null,"cover_width":1613,"cover_height":764,"account_url":null,"account_id":null,"privacy":"hidden","layout":"blog","views":28,"link":"https:\/\/imgur.com\/a\/S2VbYTE","favorite":false,"nsfw":false,"section":null,"images_count":3,"in_gallery":false,"is_ad":false,"include_album_ads":false,"is_album":true,"images":[{"id":"cme6hre","title":null,"description":null,"datetime":1597543456,"type":"image\/jpeg","animated":false,"width":1613,"height":764,"size":347300,"views":64,"bandwidth":22227200,"vote":null,"favorite":false,"nsfw":null,"section":null,"account_url":null,"account_id":null,"is_ad":false,"in_most_viral":false,"has_sound":false,"tags":[],"ad_type":0,"ad_url":"","edited":"0","in_gallery":false,"link":"https:\/\/i.imgur.com\/cme6hre.jpg"},{"id":"FJcR9Cf","title":null,"description":null,"datetime":1597543457,"type":"image\/jpeg","animated":false,"width":1613,"height":764,"size":340795,"views":45,"bandwidth":15335775,"vote":null,"favorite":false,"nsfw":null,"section":null,"account_url":null,"account_id":null,"is_ad":false,"in_most_viral":false,"has_sound":false,"tags":[],"ad_type":0,"ad_url":"","edited":"0","in_gallery":false,"link":"https:\/\/i.imgur.com\/FJcR9Cf.jpg"},{"id":"9XfX2t4","title":null,"description":null,"datetime":1597543457,"type":"image\/jpeg","animated":false,"width":2016,"height":954,"size":557826,"views":41,"bandwidth":22870866,"vote":null,"favorite":false,"nsfw":null,"section":null,"account_url":null,"account_id":null,"is_ad":false,"in_most_viral":false,"has_sound":false,"tags":[],"ad_type":0,"ad_url":"","edited":"0","in_gallery":false,"link":"https:\/\/i.imgur.com\/9XfX2t4.jpg"}],"ad_config":{"safeFlags":["not_in_gallery","share"],"highRiskFlags":[],"unsafeFlags":["sixth_mod_unsafe"],"wallUnsafeFlags":[],"showsAds":false,"showAdLevel":1}},"success":true,"status":200}
18:46:28.346 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper - Writing https://i.imgur.com/cme6hre.jpg to file
18:46:28.347 [pool-2-thread-2] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [!] Deleting existing file../../../..
18:46:28.348 [pool-2-thread-2] INFO  com.rarchives.ripme.ripper.DownloadFileThread -     Downloading file: https://i.imgur.com/cme6hre.jpg Retry #1
18:46:28.356 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper - Writing https://i.imgur.com/FJcR9Cf.jpg to file
18:46:28.356 [pool-2-thread-3] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [!] Deleting existing file../../../..
18:46:28.357 [pool-2-thread-3] INFO  com.rarchives.ripme.ripper.DownloadFileThread -     Downloading file: https://i.imgur.com/FJcR9Cf.jpg Retry #1
18:46:28.362 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper - Writing https://i.imgur.com/9XfX2t4.jpg to file
https://www.reddit.com/r/hardwareswap/comments/ibsq0d/usatx_h_dell_g3_gaming_laptop_16gb_ram_500gb_ssd/
18:46:28.363 [pool-2-thread-4] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [!] Deleting existing file../../../..
18:46:28.363 [pool-2-thread-4] INFO  com.rarchives.ripme.ripper.DownloadFileThread -     Downloading file: https://i.imgur.com/9XfX2t4.jpg Retry #1
18:46:28.375 [pool-2-thread-1] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [+] Saved https://i.imgur.com/eH3FtPo.jpg as ../../..
18:46:28.449 [pool-2-thread-2] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [+] Saved https://i.imgur.com/cme6hre.jpg as ../../../..
18:46:28.462 [pool-2-thread-3] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [+] Saved https://i.imgur.com/FJcR9Cf.jpg as ../../../..
18:46:28.492 [pool-2-thread-4] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [+] Saved https://i.imgur.com/9XfX2t4.jpg as ../../../..
18:46:28.795 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for www.reddit.com
18:46:28.795 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for reddit.com
https://www.reddit.com/r/hardwareswap/comments/iakdc0/usatx_h_dell_g3_gaming_laptop_16gb_ram_500gb_ssd/
18:46:29.011 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for www.reddit.com
18:46:29.011 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for reddit.com
18:46:29.110 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper -     Retrieving https://imgur.com/a/p930b3n/all
18:46:29.182 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper - {"data":{"id":"p930b3n","title":null,"description":null,"datetime":1597543479,"cover":"UlKHCWC","cover_edited":null,"cover_width":1613,"cover_height":764,"account_url":null,"account_id":null,"privacy":"hidden","layout":"blog","views":21,"link":"https:\/\/imgur.com\/a\/p930b3n","favorite":false,"nsfw":false,"section":null,"images_count":3,"in_gallery":false,"is_ad":false,"include_album_ads":false,"is_album":true,"images":[{"id":"UlKHCWC","title":null,"description":null,"datetime":1597543480,"type":"image\/jpeg","animated":false,"width":1613,"height":764,"size":438224,"views":96,"bandwidth":42069504,"vote":null,"favorite":false,"nsfw":null,"section":null,"account_url":null,"account_id":null,"is_ad":false,"in_most_viral":false,"has_sound":false,"tags":[],"ad_type":0,"ad_url":"","edited":"0","in_gallery":false,"link":"https:\/\/i.imgur.com\/UlKHCWC.jpg"},{"id":"8iXhaCI","title":null,"description":null,"datetime":1597543481,"type":"image\/jpeg","animated":false,"width":1613,"height":764,"size":109189,"views":88,"bandwidth":9608632,"vote":null,"favorite":false,"nsfw":null,"section":null,"account_url":null,"account_id":null,"is_ad":false,"in_most_viral":false,"has_sound":false,"tags":[],"ad_type":0,"ad_url":"","edited":"0","in_gallery":false,"link":"https:\/\/i.imgur.com\/8iXhaCI.jpg"},{"id":"uwY3dMM","title":null,"description":null,"datetime":1597543481,"type":"image\/jpeg","animated":false,"width":2016,"height":954,"size":511819,"views":82,"bandwidth":41969158,"vote":null,"favorite":false,"nsfw":null,"section":null,"account_url":null,"account_id":null,"is_ad":false,"in_most_viral":false,"has_sound":false,"tags":[],"ad_type":0,"ad_url":"","edited":"0","in_gallery":false,"link":"https:\/\/i.imgur.com\/uwY3dMM.jpg"}],"ad_config":{"safeFlags":["not_in_gallery","share"],"highRiskFlags":[],"unsafeFlags":["sixth_mod_unsafe"],"wallUnsafeFlags":[],"showsAds":false,"showAdLevel":1}},"success":true,"status":200}
18:46:29.244 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper - Writing https://i.imgur.com/UlKHCWC.jpg to file
18:46:29.245 [pool-2-thread-5] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [!] Deleting existing file../../../..
18:46:29.245 [pool-2-thread-5] INFO  com.rarchives.ripme.ripper.DownloadFileThread -     Downloading file: https://i.imgur.com/UlKHCWC.jpg Retry #1
18:46:29.268 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper - Writing https://i.imgur.com/8iXhaCI.jpg to file
18:46:29.268 [pool-2-thread-6] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [!] Deleting existing file../../../..
18:46:29.269 [pool-2-thread-6] INFO  com.rarchives.ripme.ripper.DownloadFileThread -     Downloading file: https://i.imgur.com/8iXhaCI.jpg Retry #1
18:46:29.275 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper - Writing https://i.imgur.com/uwY3dMM.jpg to file
https://www.reddit.com/r/hardwareswap/comments/iakbcp/usatx_h_asus_nvidia_gtx_1650_4gb_w_paypal/
18:46:29.275 [pool-2-thread-7] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [!] Deleting existing file../../../..
18:46:29.276 [pool-2-thread-7] INFO  com.rarchives.ripme.ripper.DownloadFileThread -     Downloading file: https://i.imgur.com/uwY3dMM.jpg Retry #1
18:46:29.282 [pool-2-thread-5] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [+] Saved https://i.imgur.com/UlKHCWC.jpg as ../../../..
18:46:29.294 [pool-2-thread-6] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [+] Saved https://i.imgur.com/8iXhaCI.jpg as ../../../..
18:46:29.322 [pool-2-thread-7] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [+] Saved https://i.imgur.com/uwY3dMM.jpg as ../../../..
18:46:29.541 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for www.reddit.com
18:46:29.542 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for reddit.com
https://www.reddit.com/r/SecurityClearance/comments/beb6nu/timeline_on_incident_reports_in_jpas/
18:46:29.736 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for www.reddit.com
18:46:29.737 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for reddit.com
https://www.reddit.com/r/hardwareswap/comments/av8h4z/usatxh_evga_980ti_sc_w_paypal_local_cash/
18:46:29.908 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for www.reddit.com
18:46:29.909 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for reddit.com
https://www.reddit.com/r/hardwareswap/comments/av8djw/usatxh_evga_980ti_sc_w_paypal/
18:46:30.063 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for www.reddit.com
18:46:30.064 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for reddit.com
18:46:32.138 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for www.reddit.com
18:46:32.139 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for reddit.com
https://www.reddit.com/r/hardwareswap/comments/av89kz/usatxh_evga_980ti_sc_w_paypal/
18:46:32.674 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for www.reddit.com
18:46:32.674 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for reddit.com
18:46:32.756 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper -     Retrieving https://imgur.com/a/NLiIP/all
18:46:32.823 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper - {"data":{"id":"NLiIP","title":null,"description":null,"datetime":1505291249,"cover":"kZkd14R","cover_edited":null,"cover_width":960,"cover_height":540,"account_url":null,"account_id":null,"privacy":"public","layout":"blog","views":17,"link":"https:\/\/imgur.com\/a\/NLiIP","favorite":false,"nsfw":false,"section":null,"images_count":1,"in_gallery":false,"is_ad":false,"include_album_ads":false,"is_album":true,"images":[{"id":"kZkd14R","title":null,"description":null,"datetime":1505291249,"type":"image\/jpeg","animated":false,"width":960,"height":540,"size":139063,"views":8,"bandwidth":1112504,"vote":null,"favorite":false,"nsfw":null,"section":null,"account_url":null,"account_id":null,"is_ad":false,"in_most_viral":false,"has_sound":false,"tags":[],"ad_type":0,"ad_url":"","edited":"0","in_gallery":false,"link":"https:\/\/i.imgur.com\/kZkd14R.jpg"}],"ad_config":{"safeFlags":["not_in_gallery","share"],"highRiskFlags":[],"unsafeFlags":["sixth_mod_unsafe"],"wallUnsafeFlags":[],"showsAds":false,"showAdLevel":1}},"success":true,"status":200}
18:46:32.840 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper - Writing https://i.imgur.com/kZkd14R.jpg to file
https://www.reddit.com/r/hardwareswap/comments/6zt9ms/usatxh_dell_inspiron_5xxx_series_gaming_laptop_w/
18:46:32.841 [pool-2-thread-8] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [!] Deleting existing file../../../..
18:46:32.841 [pool-2-thread-8] INFO  com.rarchives.ripme.ripper.DownloadFileThread -     Downloading file: https://i.imgur.com/kZkd14R.jpg Retry #1
18:46:32.871 [pool-2-thread-8] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [+] Saved https://i.imgur.com/kZkd14R.jpg as ../../../..
18:46:33.008 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for www.reddit.com
18:46:33.009 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for reddit.com
18:46:33.089 [Thread-3] ERROR com.rarchives.ripme.ripper.AbstractRipper - [!] Error creating save file path for description 'https://www.reddit.com/user/Travis4687/':
java.nio.file.NoSuchFileException: /home/nathan/Downloads/rips/reddit_user_Travis4687/6zt9ms_[USA-TX][H] Dell Inspiron 5XXX Series Gaming Laptop [W] Local Cash / Paypal.html
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:92) ~[?:?]
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111) ~[?:?]
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:116) ~[?:?]
	at sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:219) ~[?:?]
	at java.nio.file.spi.FileSystemProvider.newOutputStream(FileSystemProvider.java:478) ~[?:?]
	at java.nio.file.Files.newOutputStream(Files.java:224) ~[?:?]
	at com.rarchives.ripme.ripper.rippers.RedditRipper.saveText(RedditRipper.java:297) [ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at com.rarchives.ripme.ripper.rippers.RedditRipper.getAndParseAndReturnNext(RedditRipper.java:123) [ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at com.rarchives.ripme.ripper.rippers.RedditRipper.rip(RedditRipper.java:90) [ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at com.rarchives.ripme.ripper.AbstractRipper.run(AbstractRipper.java:627) [ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at java.lang.Thread.run(Thread.java:832) [?:?]
https://www.reddit.com/r/Persona5/comments/6gxwxc/just_started_ng_confused_about_something/
18:46:33.171 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for www.reddit.com
18:46:33.171 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for reddit.com
https://www.reddit.com/r/KingdomHearts/comments/66mpk8/kh2_still_issues_with_the_collection_safe_to_buy/
18:46:33.341 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for www.reddit.com
18:46:33.341 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for reddit.com
https://www.reddit.com/r/KingdomHearts/comments/66mnol/kh125_is_the_collection_safe_to_buy_been_a_few/
18:46:33.553 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for www.reddit.com
18:46:33.553 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for reddit.com
https://www.reddit.com/r/hardwareswap/comments/66lmb1/usatx_w_core_i73770_h_paypal/
18:46:33.711 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for www.reddit.com
18:46:33.711 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for reddit.com
https://www.reddit.com/r/hardwareswap/comments/66lk7u/usatx_w_core_i73770_non_k_or_xeon_e312405_v2_h/
18:46:33.871 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for www.reddit.com
18:46:33.871 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for reddit.com
18:46:34.039 [Thread-3] ERROR com.rarchives.ripme.ripper.AbstractRipper - [!] Error creating save file path for description 'https://www.reddit.com/user/Travis4687/':
java.nio.file.NoSuchFileException: /home/nathan/Downloads/rips/reddit_user_Travis4687/66lk7u_[USA-TX] [W] Core i7-3770 (Non K) or Xeon E3-1240/5 V2 [H] Paypal.html
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:92) ~[?:?]
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111) ~[?:?]
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:116) ~[?:?]
	at sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:219) ~[?:?]
	at java.nio.file.spi.FileSystemProvider.newOutputStream(FileSystemProvider.java:478) ~[?:?]
	at java.nio.file.Files.newOutputStream(Files.java:224) ~[?:?]
	at com.rarchives.ripme.ripper.rippers.RedditRipper.saveText(RedditRipper.java:297) [ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at com.rarchives.ripme.ripper.rippers.RedditRipper.getAndParseAndReturnNext(RedditRipper.java:123) [ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at com.rarchives.ripme.ripper.rippers.RedditRipper.rip(RedditRipper.java:90) [ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at com.rarchives.ripme.ripper.AbstractRipper.run(AbstractRipper.java:627) [ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at java.lang.Thread.run(Thread.java:832) [?:?]
https://www.reddit.com/r/PaxPassExchange/comments/5qggz9/wtb_south_sat_or_sun_tickets/
18:46:34.210 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for www.reddit.com
18:46:34.210 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for reddit.com
18:46:36.273 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for www.reddit.com
18:46:36.274 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for reddit.com
18:46:36.497 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper -     Retrieving http://imgur.com/a/tobeL/all
18:46:36.567 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper - {"data":{"id":"tobeL","title":null,"description":null,"datetime":1464736230,"cover":"6VktOQG","cover_edited":null,"cover_width":2048,"cover_height":1152,"account_url":null,"account_id":null,"privacy":"public","layout":"blog","views":47,"link":"https:\/\/imgur.com\/a\/tobeL","favorite":false,"nsfw":false,"section":null,"images_count":3,"in_gallery":false,"is_ad":false,"include_album_ads":false,"is_album":true,"images":[{"id":"6VktOQG","title":null,"description":null,"datetime":1464736231,"type":"image\/jpeg","animated":false,"width":2048,"height":1152,"size":130887,"views":4,"bandwidth":523548,"vote":null,"favorite":false,"nsfw":null,"section":null,"account_url":null,"account_id":null,"is_ad":false,"in_most_viral":false,"has_sound":false,"tags":[],"ad_type":0,"ad_url":"","edited":"0","in_gallery":false,"link":"https:\/\/i.imgur.com\/6VktOQG.jpg"},{"id":"XjNFKQE","title":null,"description":null,"datetime":1464736235,"type":"image\/jpeg","animated":false,"width":2048,"height":1152,"size":258410,"views":4,"bandwidth":1033640,"vote":null,"favorite":false,"nsfw":null,"section":null,"account_url":null,"account_id":null,"is_ad":false,"in_most_viral":false,"has_sound":false,"tags":[],"ad_type":0,"ad_url":"","edited":"0","in_gallery":false,"link":"https:\/\/i.imgur.com\/XjNFKQE.jpg"},{"id":"xkUvMqm","title":null,"description":null,"datetime":1464736239,"type":"image\/jpeg","animated":false,"width":2048,"height":1152,"size":216555,"views":4,"bandwidth":866220,"vote":null,"favorite":false,"nsfw":null,"section":null,"account_url":null,"account_id":null,"is_ad":false,"in_most_viral":false,"has_sound":false,"tags":[],"ad_type":0,"ad_url":"","edited":"0","in_gallery":false,"link":"https:\/\/i.imgur.com\/xkUvMqm.jpg"}],"ad_config":{"safeFlags":["not_in_gallery","share"],"highRiskFlags":[],"unsafeFlags":["sixth_mod_unsafe"],"wallUnsafeFlags":[],"showsAds":false,"showAdLevel":1}},"success":true,"status":200}
18:46:36.572 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper - Writing https://i.imgur.com/6VktOQG.jpg to file
18:46:36.572 [pool-2-thread-9] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [!] Deleting existing file../../../..
18:46:36.573 [pool-2-thread-9] INFO  com.rarchives.ripme.ripper.DownloadFileThread -     Downloading file: https://i.imgur.com/6VktOQG.jpg Retry #1
18:46:36.576 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper - Writing https://i.imgur.com/XjNFKQE.jpg to file
18:46:36.577 [pool-2-thread-10] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [!] Deleting existing file../../../..
18:46:36.578 [pool-2-thread-10] INFO  com.rarchives.ripme.ripper.DownloadFileThread -     Downloading file: https://i.imgur.com/XjNFKQE.jpg Retry #1
18:46:36.583 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper - Writing https://i.imgur.com/xkUvMqm.jpg to file
https://www.reddit.com/r/hardwareswap/comments/4ly54k/usatx_h_evga_gtx_970_ftw_edition_w_backplate_w/
18:46:36.584 [pool-2-thread-1] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [!] Deleting existing file../../../..
18:46:36.584 [pool-2-thread-1] INFO  com.rarchives.ripme.ripper.DownloadFileThread -     Downloading file: https://i.imgur.com/xkUvMqm.jpg Retry #1
18:46:36.601 [pool-2-thread-9] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [+] Saved https://i.imgur.com/6VktOQG.jpg as ../../../..
18:46:36.666 [pool-2-thread-1] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [+] Saved https://i.imgur.com/xkUvMqm.jpg as ../../../..
18:46:36.670 [pool-2-thread-10] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [+] Saved https://i.imgur.com/XjNFKQE.jpg as ../../../..
18:46:36.895 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for www.reddit.com
18:46:36.896 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for reddit.com
18:46:37.001 [Thread-3] ERROR com.rarchives.ripme.ripper.AbstractRipper - [!] Error creating save file path for description 'https://www.reddit.com/user/Travis4687/':
java.nio.file.NoSuchFileException: /home/nathan/Downloads/rips/reddit_user_Travis4687/4ly54k_[USA-TX] [H] EVGA GTX 970 FTW Edition w/ Backplate [W] PayPal.html
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:92) ~[?:?]
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111) ~[?:?]
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:116) ~[?:?]
	at sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:219) ~[?:?]
	at java.nio.file.spi.FileSystemProvider.newOutputStream(FileSystemProvider.java:478) ~[?:?]
	at java.nio.file.Files.newOutputStream(Files.java:224) ~[?:?]
	at com.rarchives.ripme.ripper.rippers.RedditRipper.saveText(RedditRipper.java:297) [ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at com.rarchives.ripme.ripper.rippers.RedditRipper.getAndParseAndReturnNext(RedditRipper.java:123) [ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at com.rarchives.ripme.ripper.rippers.RedditRipper.rip(RedditRipper.java:90) [ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at com.rarchives.ripme.ripper.AbstractRipper.run(AbstractRipper.java:627) [ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at java.lang.Thread.run(Thread.java:832) [?:?]
18:46:39.003 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for www.reddit.com
18:46:39.004 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for reddit.com
18:46:39.775 [Thread-3] ERROR com.rarchives.ripme.utils.RipUtils - [!] Unable to rip URL: https://www.reddit.com/r/hardwareswap/comments/3j3igu/usatxh_asus_strix_gtx_960_oc_2gb_gddr5_w_paypal/
18:46:39.776 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper -     Retrieving http://imgur.com/a/BpAH3/all
18:46:39.857 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper - {"data":{"id":"BpAH3","title":null,"description":null,"datetime":1441833316,"cover":"mt5nmTd","cover_edited":null,"cover_width":465,"cover_height":826,"account_url":null,"account_id":null,"privacy":"public","layout":"blog","views":0,"link":"https:\/\/imgur.com\/a\/BpAH3","favorite":false,"nsfw":false,"section":null,"images_count":3,"in_gallery":false,"is_ad":false,"include_album_ads":false,"is_album":true,"images":[{"id":"mt5nmTd","title":null,"description":null,"datetime":1441833251,"type":"image\/jpeg","animated":false,"width":465,"height":826,"size":77019,"views":4,"bandwidth":308076,"vote":null,"favorite":false,"nsfw":null,"section":null,"account_url":null,"account_id":null,"is_ad":false,"in_most_viral":false,"has_sound":false,"tags":[],"ad_type":0,"ad_url":"","edited":"0","in_gallery":false,"link":"https:\/\/i.imgur.com\/mt5nmTd.jpg"},{"id":"hClJbVF","title":null,"description":null,"datetime":1441833259,"type":"image\/jpeg","animated":false,"width":826,"height":465,"size":76130,"views":4,"bandwidth":304520,"vote":null,"favorite":false,"nsfw":null,"section":null,"account_url":null,"account_id":null,"is_ad":false,"in_most_viral":false,"has_sound":false,"tags":[],"ad_type":0,"ad_url":"","edited":"0","in_gallery":false,"link":"https:\/\/i.imgur.com\/hClJbVF.jpg"},{"id":"BA0xIeJ","title":null,"description":null,"datetime":1441833263,"type":"image\/jpeg","animated":false,"width":826,"height":465,"size":84758,"views":4,"bandwidth":339032,"vote":null,"favorite":false,"nsfw":null,"section":null,"account_url":null,"account_id":null,"is_ad":false,"in_most_viral":false,"has_sound":false,"tags":[],"ad_type":0,"ad_url":"","edited":"0","in_gallery":false,"link":"https:\/\/i.imgur.com\/BA0xIeJ.jpg"}],"ad_config":{"safeFlags":["not_in_gallery","share"],"highRiskFlags":[],"unsafeFlags":["sixth_mod_unsafe"],"wallUnsafeFlags":[],"showsAds":false,"showAdLevel":1}},"success":true,"status":200}
18:46:39.862 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper - Writing https://i.imgur.com/mt5nmTd.jpg to file
18:46:39.862 [pool-2-thread-2] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [!] Deleting existing file../../../..
18:46:39.862 [pool-2-thread-2] INFO  com.rarchives.ripme.ripper.DownloadFileThread -     Downloading file: https://i.imgur.com/mt5nmTd.jpg Retry #1
18:46:39.869 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper - Writing https://i.imgur.com/hClJbVF.jpg to file
18:46:39.870 [pool-2-thread-3] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [!] Deleting existing file../../../..
18:46:39.870 [pool-2-thread-3] INFO  com.rarchives.ripme.ripper.DownloadFileThread -     Downloading file: https://i.imgur.com/hClJbVF.jpg Retry #1
18:46:39.880 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper - Writing https://i.imgur.com/BA0xIeJ.jpg to file
https://www.reddit.com/r/hardwareswap/comments/3kashp/usatx_h_crucial_bx100_250gb_sata_iii_ssd_6gbps_w/
18:46:39.880 [pool-2-thread-4] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [!] Deleting existing file../../../..
18:46:39.881 [pool-2-thread-4] INFO  com.rarchives.ripme.ripper.DownloadFileThread -     Downloading file: https://i.imgur.com/BA0xIeJ.jpg Retry #1
18:46:39.891 [pool-2-thread-2] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [+] Saved https://i.imgur.com/mt5nmTd.jpg as ../../../..
18:46:39.896 [pool-2-thread-3] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [+] Saved https://i.imgur.com/hClJbVF.jpg as ../../../..
18:46:39.911 [pool-2-thread-4] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [+] Saved https://i.imgur.com/BA0xIeJ.jpg as ../../../..
18:46:40.758 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for www.reddit.com
18:46:40.759 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for reddit.com
18:46:40.860 [Thread-3] ERROR com.rarchives.ripme.ripper.AbstractRipper - [!] Error creating save file path for description 'https://www.reddit.com/user/Travis4687/':
java.nio.file.NoSuchFileException: /home/nathan/Downloads/rips/reddit_user_Travis4687/3kashp_[USA-TX] [H] Crucial BX100 250GB SATA III SSD 6Gbp/s [W] PayPal $75 Shipped.html
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:92) ~[?:?]
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111) ~[?:?]
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:116) ~[?:?]
	at sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:219) ~[?:?]
	at java.nio.file.spi.FileSystemProvider.newOutputStream(FileSystemProvider.java:478) ~[?:?]
	at java.nio.file.Files.newOutputStream(Files.java:224) ~[?:?]
	at com.rarchives.ripme.ripper.rippers.RedditRipper.saveText(RedditRipper.java:297) [ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at com.rarchives.ripme.ripper.rippers.RedditRipper.getAndParseAndReturnNext(RedditRipper.java:123) [ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at com.rarchives.ripme.ripper.rippers.RedditRipper.rip(RedditRipper.java:90) [ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at com.rarchives.ripme.ripper.AbstractRipper.run(AbstractRipper.java:627) [ripme-2.1.3-18-ea5ffdba.jar:2.1.3-18-ea5ffdba]
	at java.lang.Thread.run(Thread.java:832) [?:?]
18:46:40.860 [Thread-3] ERROR com.rarchives.ripme.utils.RipUtils - [!] Unable to rip URL: https://www.asus.com/us/Graphics_Cards/STRIXGTX960DC2OC2GD5/
18:46:40.860 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper -     Retrieving http://imgur.com/a/4rd4S/all
18:46:40.932 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper - {"data":{"id":"4rd4S","title":"960 STRIX","description":null,"datetime":1441035521,"cover":"BBWUjrc","cover_edited":null,"cover_width":3715,"cover_height":2089,"account_url":null,"account_id":null,"privacy":"public","layout":"blog","views":0,"link":"https:\/\/imgur.com\/a\/4rd4S","favorite":false,"nsfw":false,"section":null,"images_count":4,"in_gallery":false,"is_ad":false,"include_album_ads":false,"is_album":true,"images":[{"id":"BBWUjrc","title":null,"description":null,"datetime":1441035170,"type":"image\/jpeg","animated":false,"width":3715,"height":2089,"size":568837,"views":4,"bandwidth":2275348,"vote":null,"favorite":false,"nsfw":null,"section":null,"account_url":null,"account_id":null,"is_ad":false,"in_most_viral":false,"has_sound":false,"tags":[],"ad_type":0,"ad_url":"","edited":"0","in_gallery":false,"link":"https:\/\/i.imgur.com\/BBWUjrc.jpg"},{"id":"OnviMQi","title":null,"description":null,"datetime":1441035276,"type":"image\/jpeg","animated":false,"width":3715,"height":2089,"size":500200,"views":4,"bandwidth":2000800,"vote":null,"favorite":false,"nsfw":null,"section":null,"account_url":null,"account_id":null,"is_ad":false,"in_most_viral":false,"has_sound":false,"tags":[],"ad_type":0,"ad_url":"","edited":"0","in_gallery":false,"link":"https:\/\/i.imgur.com\/OnviMQi.jpg"},{"id":"UVepKHT","title":null,"description":null,"datetime":1441035387,"type":"image\/jpeg","animated":false,"width":3715,"height":2089,"size":733573,"views":4,"bandwidth":2934292,"vote":null,"favorite":false,"nsfw":null,"section":null,"account_url":null,"account_id":null,"is_ad":false,"in_most_viral":false,"has_sound":false,"tags":[],"ad_type":0,"ad_url":"","edited":"0","in_gallery":false,"link":"https:\/\/i.imgur.com\/UVepKHT.jpg"},{"id":"ma4j7ES","title":null,"description":null,"datetime":1441035495,"type":"image\/jpeg","animated":false,"width":3715,"height":2089,"size":558214,"views":4,"bandwidth":2232856,"vote":null,"favorite":false,"nsfw":null,"section":null,"account_url":null,"account_id":null,"is_ad":false,"in_most_viral":false,"has_sound":false,"tags":[],"ad_type":0,"ad_url":"","edited":"0","in_gallery":false,"link":"https:\/\/i.imgur.com\/ma4j7ES.jpg"}],"ad_config":{"safeFlags":["not_in_gallery","share"],"highRiskFlags":[],"unsafeFlags":["sixth_mod_unsafe"],"wallUnsafeFlags":[],"showsAds":false,"showAdLevel":1}},"success":true,"status":200}
18:46:40.957 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper - Writing https://i.imgur.com/BBWUjrc.jpg to file
18:46:40.958 [pool-2-thread-5] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [!] Deleting existing file../../../..
18:46:40.959 [pool-2-thread-5] INFO  com.rarchives.ripme.ripper.DownloadFileThread -     Downloading file: https://i.imgur.com/BBWUjrc.jpg Retry #1
18:46:40.968 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper - Writing https://i.imgur.com/OnviMQi.jpg to file
18:46:40.968 [pool-2-thread-6] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [!] Deleting existing file../../../..
18:46:40.969 [pool-2-thread-6] INFO  com.rarchives.ripme.ripper.DownloadFileThread -     Downloading file: https://i.imgur.com/OnviMQi.jpg Retry #1
18:46:40.972 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper - Writing https://i.imgur.com/UVepKHT.jpg to file
18:46:40.972 [pool-2-thread-7] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [!] Deleting existing file../../../..
18:46:40.973 [pool-2-thread-7] INFO  com.rarchives.ripme.ripper.DownloadFileThread -     Downloading file: https://i.imgur.com/UVepKHT.jpg Retry #1
18:46:40.977 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper - Writing https://i.imgur.com/ma4j7ES.jpg to file
https://www.reddit.com/r/hardwareswap/comments/3j3igu/usatxh_asus_strix_gtx_960_oc_2gb_gddr5_w_paypal/
18:46:40.978 [pool-2-thread-8] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [!] Deleting existing file../../../..
18:46:40.978 [pool-2-thread-8] INFO  com.rarchives.ripme.ripper.DownloadFileThread -     Downloading file: https://i.imgur.com/ma4j7ES.jpg Retry #1
18:46:41.006 [pool-2-thread-5] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [+] Saved https://i.imgur.com/BBWUjrc.jpg as ../../../..
18:46:41.025 [pool-2-thread-6] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [+] Saved https://i.imgur.com/OnviMQi.jpg as ../../../..
18:46:41.039 [pool-2-thread-7] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [+] Saved https://i.imgur.com/UVepKHT.jpg as ../../../..
18:46:41.127 [pool-2-thread-8] INFO  com.rarchives.ripme.ripper.DownloadFileThread - [+] Saved https://i.imgur.com/ma4j7ES.jpg as ../../../..
18:46:41.198 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for www.reddit.com
18:46:41.199 [Thread-3] INFO  com.rarchives.ripme.utils.Http - Trying to load cookies from config for reddit.com
18:46:43.317 [Thread-3] INFO  com.rarchives.ripme.ripper.AbstractRipper -    Rip completed!
```

--- a/src/main/java/com/rarchives/ripme/ripper/rippers/RedditRipper.java
+++ b/src/main/java/com/rarchives/ripme/ripper/rippers/RedditRipper.java
@@ -292,7 +292,7 @@ public class RedditRipper extends AlbumRipper {
         try {
             saveFileAs = Utils.getPath(workingDir
                     + "/"
-                    + id + "_" + title
+                    + id + "_" + Utils.filesystemSafe(title)
                     + ".html");
             OutputStream out = Files.newOutputStream(saveFileAs);
             out.write(html.getBytes());
@@ -411,12 +411,12 @@ public class RedditRipper extends AlbumRipper {
             if (m.matches()) {
                 // It's from reddituploads. Assume .jpg extension.
                 String savePath = this.workingDir + "/";
-                savePath += id + "-" + m.group(1) + title + ".jpg";
+                savePath += id + "-" + m.group(1) + Utils.filesystemSafe(title) + ".jpg";
                 addURLToDownload(urls.get(0), Utils.getPath(savePath));
             }
             if (url.contains("v.redd.it")) {
                 String savePath = this.workingDir + "/";
-                savePath += id + "-" + url.split("/")[3] + title + ".mp4";
+                savePath += id + "-" + url.split("/")[3] + Utils.filesystemSafe(title) + ".mp4";
                 URL urlToDownload = parseRedditVideoMPD(urls.get(0).toExternalForm());
                 if (urlToDownload != null) {
                     LOGGER.info("url: " + urlToDownload + " file: " + savePath);
@@ -424,7 +424,7 @@ public class RedditRipper extends AlbumRipper {
                 }
             }
             else {
-                addURLToDownload(urls.get(0), id + title, "", theUrl, null);
+                addURLToDownload(urls.get(0), Utils.filesystemSafe(id + title), "", theUrl, null);
             }
         } else if (urls.size() > 1) {
             for (int i = 0; i < urls.size(); i++) {

